darwin/importer/formats/labelbox.py

```diff
@@ -89,10 +86,10 @@ def _convert(file_data: Dict[str, Any], path) -> AnnotationFile:
     )
 
 
+# We ignore the Any | None warning here because the schema has been validated and we know
+# we have the values with the correct types
 def _convert_label_objects(obj: Dict[str, Any]) -> Annotation:
-    title: Optional[str] = obj.get("title")
-    if not title:
-        raise ValueError(f"LabelBox objects must have a title: {obj}")
+    title: str = obj.get("title")  # type: ignore
```

Comment:
Would it look nicer to do `str(obj.get("title"))` instead of ignoring the types?

Reply:
Although that would work for `str(None)` it would not for the other cases. 
If you have any suggestions, I am all ears. The only ones I found were:
https://stackoverflow.com/questions/68446642/how-do-i-get-pylance-to-ignore-the-possibility-of-none

Any of them sound better then ignoring type casting, but I have not settled on one. @andreaazzini  for example is not a fan of `assert`s.

Reply:
Isn't this `title` coming from the schema that has just been validated? If so, then we know for sure that it's a string-like variable. In this case, doing `str(...)` should be pretty safe. Am I not seeing something here?

Reply:
My comment is in regards to Pylance. Pylance doesnt know there was  a validation and therefore will always report an error. It is this I am trying to address here.

Reply:
Fixed is static `cast`